packages/flame/lib/src/components/joystick/joystick_action.dart

```diff
@@ -186,18 +187,18 @@ class JoystickAction extends BaseComponent with Draggable, HasGameRef {
   }
 
   @override
-  bool onDragUpdate(int pointerId, DragUpdateDetails details) {
+  bool onDragUpdate(int pointerId, DragUpdateInfo event) {
     if (_dragging) {
       _dragPosition = gameRef.convertGlobalToLocalCoordinate(
-        details.globalPosition.toVector2(),
+        event.raw.globalPosition.toVector2(),
       );
       return true;
     }
     return false;
   }
 
   @override
-  bool onDragEnd(int pointerId, DragEndDetails p1) {
+  bool onDragEnd(_, __) {
```

Comment:
This looks weird when `_` isn't built-in to the language (especially with the `__`), is the usage of `_` recommended in the style guide?

Reply:
No strong opinion either. cc @renancaraujo

Reply:
If it is native, why does the second one need to be `__`? Maybe just me that is used to how it works in functional programming languages, but this looks more like two variables that are named `_` and `__`, than a "don't care"-operator.